src/rf/apps/workers/emr.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import logging
+
+import boto3
+from django.conf import settings
+
+from apps.core.models import LayerImage
+
+
+log = logging.getLogger(__name__)
+
+
+def create_cluster(layer):
+    """
+    Create new EMR cluster.
+    """
+    queue_name = settings.AWS_SQS_QUEUE
+    artifacts = settings.AWS_ARTIFACTS_BUCKET
+
+    client = boto3.client('emr')
+    status_queue = get_queue_url(queue_name)
+
+    logs_uri = 's3://{}'.format(settings.AWS_LOGS_BUCKET)
+    bootstrap_uri = 's3://{}/bootstrap.sh'.format(artifacts)
+
+    instances = {
+        # For debugging only.
+        # 'Ec2KeyName': 'rf-emr',
+        # 'KeepJobFlowAliveWhenNoSteps': True,
+
+        'InstanceGroups': [
+            {
+                'Name': 'Master',
+                'InstanceRole': 'MASTER',
+                'InstanceType': 'm3.xlarge',
```

Comment:
These instances gave us the best results back when we did the climate benchmarking. There's also a restricted set of instances that you can run with EMR.

If you use other instances, you'll have to modify the spark config values of the number of cores and memory per executor, which are set with the EMR launch command